js/components/operatorSignIn/complete.tsx

```diff
@@ -0,0 +1,39 @@
+import { className } from "../../util/dom";
+import { ReactElement } from "react";
+
+export const Success = ({ name }: { name: string }): ReactElement => {
+  return (
+    <div className="text-center text-4xl w-full block mt-5 p-10">
+      ✅<div>{name} signed in successfully</div>
```

Comment:
**question:** the name should also get the `fs-mask` class, right? I know we don't have FullStory yet but I see we've been adding the class in some places, so it would probably be good to be as thorough as possible with it.

Reply:
Same question pertains to the name in the `Error` component.

Reply:
062b1f7